src/strands_tools/slack.py

```diff
@@ -386,12 +387,14 @@ def _process_message(self, event):
                 logger.info(f"Skipping message - does not contain tag: {listen_only_tag}")
                 return
 
-            # Process with agent
-            response = agent(
-                f"[Channel: {channel_id}] User {user} says: {text}",
-                system_prompt=f"{SLACK_SYSTEM_PROMPT}\n\nEvent Context:\nCurrent: {json.dumps(event)}{event_context}",
+            # Refresh the system prompt with latest context handled from Slack events
+            agent.system_prompt = (
```

Comment:
We recently removed the support of ephemeral system prompt changes ^^

Without the system prompt updates slack agents is unable to track the thread's context (since the `event_context` is populated from slack events)